huntflow_api_client/entities/applicant_on_vacancy.py

```diff
@@ -0,0 +1,102 @@
+from huntflow_api_client.entities.base import BaseEntity, ListEntityMixin
+from huntflow_api_client.models.request.applicant_on_vacancy import (
+    AddApplicantToVacancyRequest,
+    ApplicantVacancySplitRequest,
+    ChangeVacancyApplicantStatusRequest,
+)
+from huntflow_api_client.models.response.applicant_on_vacancy import (
+    AddApplicantToVacancyResponse,
+    ApplicantVacancySplitResponse,
+    RejectionReasonsListResponse,
+    VacancyStatusesResponse,
+)
+
+
+class ApplicantOnVacancy(BaseEntity, ListEntityMixin):
+    async def list(self, account_id: int) -> VacancyStatusesResponse:
+        """
+        API method reference
+            https://api.huntflow.ai/v2/docs#get-/accounts/-account_id-/vacancies/statuses
+
+        :param account_id: Organization ID
+        :return: List of available applicant on vacancy statuses (stages)
+        """
+        response = await self._api.request("GET", f"/accounts/{account_id}/vacancies/statuses")
+        return VacancyStatusesResponse.parse_obj(response.json())
```

Comment:
The same as "rejection reason": it looks more reasonable to place it in `ApplicantOnVacancyStatus.list`. I can think that `ApplicantOnVacancy.list` should return some list of applicants and vacancies